examples/trt_llama.yaml

```diff
@@ -15,7 +15,6 @@ launcher:
 backend:
   device: cuda
   device_ids: 0
-  no_weights: true
```

Comment:
no weights does not seem to be supported by the trt backend

Reply:
it used to be, I don't remember when it was removed tbh.